modules/webrtc/webrtc_multiplayer_peer.cpp

```diff
@@ -165,30 +169,51 @@ void WebRTCMultiplayerPeer::_find_next_peer() {
 			++E;
 			continue;
 		}
+		int idx = 0;
 		for (const Ref<WebRTCDataChannel> &F : E->value->channels) {
 			if (F->get_available_packet_count()) {
+				next_packet_channel = idx;
 				next_packet_peer = E->key;
 				return;
 			}
+			idx++;
 		}
 		if (E->key == (int)next_packet_peer) {
 			break;
 		}
 		++E;
 	}
 	// No packet found
+	next_packet_channel = 0;
 	next_packet_peer = 0;
 }
 
 MultiplayerPeer::ConnectionStatus WebRTCMultiplayerPeer::get_connection_status() const {
 	return connection_status;
 }
 
-Error WebRTCMultiplayerPeer::initialize(int p_self_id, bool p_server_compat, Array p_channels_config) {
+Error WebRTCMultiplayerPeer::create_server(Array p_channels_config) {
+	return _initialize(1, MODE_SERVER, p_channels_config);
+}
+
+Error WebRTCMultiplayerPeer::create_client(int p_self_id, Array p_channels_config) {
+	ERR_FAIL_COND_V_MSG(p_self_id == 1, ERR_INVALID_PARAMETER, "Clients cannot have ID of 1.");
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(p_self_id == 1, ERR_INVALID_PARAMETER, "Clients cannot have ID 1.");
```
(Or "an idea of 1", I guess.)